### PR TITLE
FOUR-8194 Renamed mostly everything to cloneButton except Modeler sin…

### PR DESCRIPTION
**Pull request by josechirivella**
…ce I wanted to avoid big changes with the other team members

* Changed the label to Clone element

## Issue & Reproduction Steps

Expected behavior: 
Label should be `Clone Element`

Actual behavior: 
Label is `Duplicate Element`

## Solution
- Did a rename on the label
- Changed the filename to cloneButton and did several renaming in functions to be more targetted to clone

## How to Test
Test the steps above

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-8194

## Code Review Checklist
- [x] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [x] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [x] This solution fixes the bug reported in the original ticket.
- [x] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [x] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [x] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [x] This ticket conforms to the PRD associated with this part of ProcessMaker.

### CODE REVIEW

**Review by caleeli — APPROVED** [review]

**Review by boliviacoca — APPROVED** [review]

File: src/components/crown/crownMultiselect/crownMultiselect.vue [hunk 1]
```diff
@@ -51,7 +51,7 @@ export default {
         {
           label: 'Duplicate Selection',
```
Comment (caleeli) [inline src/components/crown/crownMultiselect/crownMultiselect.vue:52]: This label is still saying "Duplicate"

**Review by caleeli — CHANGES_REQUESTED** [review]
Please update the label also for multi-selection

**Comment by ana-mauricio** [comment]
The fix should be modified : 
- Each word must start capital letter like the other labels "**Clone Element**"
- And the label must also be modified to the selection 
image
@josechirivella

**Review by paulosakamoto — APPROVED** [review]

**Comment by ana-mauricio** [comment]
The Fix is correct @josechirivella

**Pull request merged**